Clicking on Canvas without init causes Exception

Caused by: java.lang.NullPointerException
	at com.treasure.hunt.service.select.SelectionService.handleClickEvent(SelectionService.java:67)
	at com.treasure.hunt.view.CanvasController.onMouseLifted(CanvasController.java:102)
	... 44 more

--- a/src/main/java/com/treasure/hunt/service/select/SelectionService.java
+++ b/src/main/java/com/treasure/hunt/service/select/SelectionService.java
@@ -51,6 +51,10 @@ public class SelectionService {
 
     @SneakyThrows
     public void handleClickEvent(MouseEvent mouseEvent, PointTransformation transformation, GameManager gameManager) {
+        if (gameManager == null) {
+            return;
+        }
+
         Coordinate coordinate = transformation.revert(mouseEvent.getX(), mouseEvent.getY());
 
         if (currentGameManager != gameManager) {

